File: 1001 - 1020/1020.py
age_in_days = int(input())

years  = age_in_days // 365
months = age_in_days % 365 // 30
days   = age_in_days % 365 % 30

print('{0} ano(s)'.format(years))
print('{0} mes(es)'.format(months))
print('{0} dia(s)'.format(days))

File: 1001 - 1020/1020.c
#include<stdio.h>

int main(){
  int years, months, days;
  int age;

  scanf("%d", &age);

  years  = age / 365;
  months = age % 365 / 30;
  days   = age % 365 % 30;

  printf("%d ano(s)\n", years);
  printf("%d mes(es)\n", months);
  printf("%d dia(s)\n", days);
  return 0;
}
